core/src/main/java/org/apache/accumulo/core/metadata/StoredTabletFile.java

```diff
@@ -46,7 +49,7 @@ public class StoredTabletFile extends AbstractTabletFile<StoredTabletFile> {
   public StoredTabletFile(String metadataEntry) {
     super(new Path(metadataEntry));
     this.metadataEntry = metadataEntry;
```

Comment:
Well (in theory) we should be only ever be passing in a String that was read from metadata so that means it was already stored before so it should not need validation as it would have already passed validation when stored originally. 

When we store new files the metadata gets generated from the[ insert() ](https://github.com/apache/accumulo/blob/6a1431b773c49f422e109268dccd460661d5e02f/core/src/main/java/org/apache/accumulo/core/metadata/ReferencedTabletFile.java#L114)method of ReferencedTabletFile.

Reply:
Of course there could be a mistake in the future but as of now it's not an issue when I search the code. I took out that validate call because it was pointed out as an issue in the original Issue description #3500. 

Leaving the call the performance was 0.005 ms/op but with it removed I get 0.004 ms/op so a little better.